### PR TITLE
6674 Add Permanent Address, Temp Address, Supplies into MDOT Prefill

### PR DESCRIPTION
**Pull request by camerontesterman**
## Original Issue
[6674](https://github.com/department-of-veterans-affairs/va.gov-team/issues/6674)

## Description of change
This changes the `veteranAddress` field that is returned in the MDOT prefill to `permanentAddress` and also adds a separate field for `temporaryAddress`. This also adds the functionality to display medical devices and supplies to the MDOT form prefill model, and exposes those supplies in the prefill endpoint.

## Testing done
Tested locally

## Testing planned
None at this time - will test with DLC once their work is completed and their API is deployed in their lower environment.

## Acceptance Criteria (Definition of Done)
Prefill data is being returned correctly when you `GET /v0/in_progress_forms/MDOT`

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -669,7 +669,7 @@ GEM
     sidekiq-instrument (0.3.0)
       sidekiq (>= 4.2, < 6)
       statsd-instrument (~> 2.0, >= 2.0.4)
-    sidekiq-pro (3.7.0)
+    sidekiq-pro (3.7.1)
```
Comment (johnpaulashenfelter) [inline Gemfile.lock]: @camerontesterman we're not ready to upgrade sidekiq pro yet. Please revert that before you merge. The vets_json_schema update is clearly part of this so no worries there.

Comment (camerontesterman) [reply Gemfile.lock]: Ok, sounds good. I just pinned it to 3.7.0 to prevent that from occurring again until the platform team is ready to bump it.

**Comment by camerontesterman** [comment]
> Please revert the update to the `sidekiq-pro` gem update @camerontesterman. Would be great to have someone on your team review as well, but at a minimum revert that change and we're ok.

I just added Mahariel and Amen to take a look, and that gem is now pinned at `3.7.0`

**Comment by mr0sari0** [comment]
@camerontesterman this is the response I'm getting:

`{
    "form_data": {
        "veteran_address": {
            "street": "MILITARY ADDY 3",
            "city": "DPO",
            "country": "USA",
            "state": "AA",
            "postal_code": "22312"
        },
        "email": "test2@test1.net",
        "veteran_full_name": {
            "first": "Greg",
            "middle": "A",
            "last": "Anderson"
        },
        "gender": "M",
        "date_of_birth": "1933-04-05"
    },
    "metadata": {
        "version": 0,
        "return_url": "/veteran-information/addresses",
        "saved_at": 1584554021382,
        "expires_at": 1589738022,
        "last_updated": 1584554022
    }
}`

File: Gemfile.lock (hunk 1 above)
Comment (johnpaulashenfelter) [reply Gemfile.lock]: Thanks. We're dealing with the bigger upgrade of sidekiq separately.

**Pull request merged**